### PR TITLE
Riley.fixes and instructions

### PR DESCRIPTION
**Pull request by sp33drac3r**
Make demo work on our infrastructure
These were the changes necessary to get this demo
working on our infrastructure. What changed
since this demo was given in 2017?

  - New version of kubectl.
  - New version of go.
  - Fewer Google Cloud resources than @jessup.

These changes will allow us to continue development
of our demo in our Google Cloud Kubernetes cluster.
Other demo clusters should fork from this point.

Changes include:
  - Use standard library slice.
  - Update apiVersion: apps/v1 for all deployments.
  - Update namespace: gifinator for all deployments.
  - Our Google project ID.

### CODE REVIEW

**Comment by sp33drac3r** [comment]
More changes required to get this to build in k8s. Removing review request for now.

File: k8s/gifcreator-deployment.yaml [hunk 1]
```diff
@@ -29,7 +33,11 @@ spec:
     spec:
       containers:
         - name: gifcreator-server
+<<<<<<< HEAD
           image: gcr.io/jessup-spinnaker-test/gifcreator:d29d90c2-8001-40aa-b665-e489c2d67c74 # TODO(jessup) Ideally this is dynamic
+=======
+          image: gcr.io/celtic-hub-325419/gifcreator:d9d884f1-b66e-4a82-8706-efa4dbb5802c # TODO(jessup) Ideally this is dynamic
+>>>>>>> 6525895 (Prettier all files)
```
Comment (rjhuijsman) [inline k8s/gifcreator-deployment.yaml:40]: Oops! We should resolve these conflicts (here and below).

File: README.md [hunk 2]
```diff
@@ -23,10 +23,16 @@ go get github.com/mattn/goreman
 You will also need to install the [Google Cloud SDK](https://cloud.google.com/sdk/downloads) to build
 and deploy to Kubernetes.
 
+### Setup Google Cloud Storage
+
 You will also need to have created a Google Cloud Storage bucket, for exclusive
 use by the application. If you plan to deploy to GKE, it is suggested to
 create your bucket in the same project.
 
+Remove public access preventions and allow fine-grained access controls on the bucket you create.
```
Comment (rjhuijsman) [inline README.md:32]: Could you also drop a link to the documentation we used to create the Kubernetes service account and link it to a Google service account?

https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to

**Review by rjhuijsman — CHANGES_REQUESTED** [review]

File: k8s/gifcreator-deployment.yaml (hunk 1 above)
Comment (sp33drac3r) [reply k8s/gifcreator-deployment.yaml:40]: Oops! How did that happen? Good catch!

File: README.md (hunk 2 above)
Comment (sp33drac3r) [reply README.md:32]: Good idea. Will do

**Review by rjhuijsman — APPROVED** [review]

**Pull request merged**